src/composite/MegaChart.js

```diff
@@ -45,9 +45,11 @@
     MegaChart.prototype.publish("titleFontFamily", null, "string", "Title Font Family", null, { tags: ["Advanced"], optional: true });
     MegaChart.prototype.publish("titleFontBold", true, "boolean", "Enable Bold Title Font", null, { tags: ["Advanced"], optional: true });
     MegaChart.prototype.publish("titleBackgroundColor", null, "html-color", "Background Color", null, { tags: ["Intermediate"], optional: true });
+    MegaChart.prototype.publish("maximizedBackgroundColor", '#FFFFFF', "html-color", "Background Color while maximized", null, { tags: ["Intermediate"], optional: true });
```

Comment:
maximizedBorderShadowColor? ...but the maximized widgets need a background color so that the widget's content is visible (IOW there may be very "busy" graphics behind the maximized widget)